### PR TITLE
WK-1086: link to entity if document exists.

### PR DESCRIPTION
**Pull request by nakuzm**

### CODE REVIEW

**Comment by deborovik** [comment]
👍

**Review by deborovik — APPROVED** [review]
👍

File: src/commands/veloce/loadcontentdoc.ts [hunk 1]
```diff
@@ -13,6 +13,59 @@ Messages.importMessagesDirectory(__dirname)
...
+      }),
+      method: 'POST'
+    })) as RestResult;
+
+    if (!response.success) {
+      console.log(`Failed to link document ${docId} to object ${linkedEntityId}, error: ${response.errors.join(',')}`);
+      process.exit(255);
+    }
+  }
+  if (!skipdelete) {
```
Comment (dmnrmr) [inline src/commands/veloce/loadcontentdoc.ts:48]: This is changing the default behavior which potentially might cause problems in places where it is used. The flag should be the other way around, it should enable deletion.

Comment (nakuzm) [reply src/commands/veloce/loadcontentdoc.ts:48]: ok

**Pull request merged**